README.md

```diff
@@ -105,7 +91,7 @@ Pa11y CI has a few of its own configurations which you can set as well:
 
 ### URL configuration
 
-Each URL in your config file can be an object and specify [pa11y configurations] which override the defaults too. You do this by using an object instead of a string, and providing the URL as a `url` property on that object. This can be useful if, for example, you know that a certain URL takes a while to load or you want to check what the page looked like when the tests were run:
+A URL can be a `string` or an `object`. In its object form, aspects of the default [pa11y configurations] can be overridden per URL. For example, this allows the timeout to be increased for a slow-loading page, or to take a screenshot for a page of particular interest:
```

Comment:
What kind of object? A [Node URL](https://nodejs.org/docs/latest-v16.x/api/url.html) ?

Reply:
👍 I've tied together that line with the one that comes straight afterwards. Does that make it a bit clearer, like the previous version?